README.md

```diff
@@ -1,6 +1,8 @@
-Travis Bionic Zeroc Ice Python 3 Builder
-========================================
+Bionic Zeroc Ice Python 3 Builder
```

Comment:
Comparing with https://github.com/ome/zeroc-ice-ubuntu1804, shoudl we use `Ubuntu 18.04` instead of bionic everywhere?